<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="schuifdeurKnop">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>VD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hallsensor">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>s1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideur1Knop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>d1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="draaideur2Knop">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>d2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHalsensor">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>halsensor</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
